src/components/combobox-item/combobox-item.tsx

```diff
@@ -158,6 +165,7 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
       return;
     }
     this.selected = !this.selected;
+    this.calciteComboboxItemFocus.emit(this.el);
```

Comment:
@alisonailea let me see if I'm understanding the issue correctly. From what I understand, the problem is that the `combobox` gets focus on its text input anytime a `combobox-item` is selected. right?

I think this is because the watcher on the combobox item emits the selection event no matter if its from user input or programatic.

```
 @Watch("selected")
  selectedWatchHandler(): void {
    this.calciteComboboxItemChange.emit(this.el);
  }
```

What I was suggesting, is that we change the watcher to emit an internal event named `calciteInternalComboboxItemChange`. This event would not be documented (marked as `@internal`).

This internal event would emit on watch change and its event handler can update selection on the `combobox` but NOT focus the internal input. I think this is ideal because this event really should only fire on user interaction anyway and not programatic changes.

Then, we only emit the public `calciteComboboxItemChange` when a user input occurs. For example in onClick (and keyboard events?) we would call this event. The public event would do the same stuff as the internal event listener plus handle focusing. If we only want to focus on click and not keyboard events then we probably need that internal focus event that you created.

Does that make sense?

Reply:
Just curious, do we want the text box to get focus only on click? Or via keyboard enter/space events as well?

Reply:
after reviewing your comment and reassessing the code I realized that only the combobox has to be aware of focus at all. Since we are already handling all keypress events on combobox we can handle the click events as well. I've updated the code and removed `calciteComboboxItemFocus` completely then added a click event handler to the connectedCallback(and disconnnectedCallback) of combobox